articles/active-directory-b2c/active-directory-b2c-devquickstarts-graph-dotnet.md

```diff
@@ -17,7 +17,13 @@ ms.date: 08/07/2017
 ms.author: parakhj
 
 ---
-# Azure AD B2C: Use the Graph API
+# Azure AD B2C: Use the Azure AD Graph API
+
+[!NOTE]
+As of this time, you must use the [Azure AD Graph API](https://msdn.microsoft.com/Library/Azure/Ad/Graph/howto/azure-ad-graph-api-operations-overview?f=255&MSPPError=-2147217396) to manage B2C Tenant(s).  In the future, [Microsoft Graph](https://developer.microsoft.com/en-us/graph) will become the API.
+1. [Microsoft Graph or the Azure AD Graph](https://dev.office.com/blogs/microsoft-graph-or-azure-ad-graph)
+1. [Stack Overflow Post](https://stackoverflow.com/questions/43770376/which-graph-api-should-be-used-with-azure-ad-b2c)
+
```

Comment:
Thanks for writing this up! Could you update lines 22 to line 25 to the following (include the > characters)

>[!NOTE]
>Currently you must use the [Azure AD Graph API](https://msdn.microsoft.com/Library/Azure/Ad/Graph/howto/azure-ad-graph-api-operations-overview?f=255&MSPPError=-2147217396) to manage the users in an Azure AD B2C directory. In the future, [Microsoft Graph](https://developer.microsoft.com/en-us/graph) will become the API. Learn more [here](https://dev.office.com/blogs/microsoft-graph-or-azure-ad-graph).

Reply:
@parakhj done 😄